### DISCUSSION TITLE
Build Fails with Severity Unknown build error, 'A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)'

### DISCUSSION THREAD

**Discussion by SQ0227** [cefsharp/CefSharp#3654] [General]
Checked for similar Issue on Github and Stackoverflow. 
Didn't find a suitable answer and one similar Issue was in 2014. 

Environment: 
VS 2019 Community Edition 
**.Net Framework** 4.5.2 (Tried with 4.6.* and 4.7.2)
**Project Configuration**: Set to x86 Configuration. 

Installed Version of CefSharp (Same results with 91.1.211)
![image](https://user-images.githubusercontent.com/86592993/123610222-08038900-d81e-11eb-9f1b-881902008ff8.png)

**Issue**
Severity	Code	Description	Project	File	Line	Suppression State
Error		Unknown build error, 'A strongly-named assembly is required. (Exception from HRESULT: 0x80131044) Line 10 Position 10.'	CefSharpTest	...\source\repos\CefSharp\CefSharp\MainWindow.xaml	10	

**Main Window**
![image](https://user-images.githubusercontent.com/86592993/123610641-6597d580-d81e-11eb-8b7d-781505a0096b.png)

**Comment by amaitland** [comment]
The CefSharp assemblies are already strongly named. CefSharp.dll for example has https://github.com/cefsharp/CefSharp/blob/cefsharp/91/CefSharp/CefSharp.csproj#L26

Are you upgrading from a previous version? Which CefSharp assembly is causing the problem? 

- Please include a detailed description of what's going on (don't rely on images) 
- Please include a clear list of steps to reproduce the problem

**Reply by SQ0227** [reply]
Yes,

Issue under discussion: 

> Severity	Code	Description	Project	File	Line	Suppression State
> Error		Unknown build error, 'A strongly-named assembly is required. (Exception from HRESULT: 0x80131044) Line 10 Position 10.'	CefSharpTest	...\source\repos\CefSharp\CefSharp\MainWindow.xaml	10
	
The steps I followed to create were really simple: 
1. Created a new WPF project with .Net Framework 4.7.2, added CefSharp (**91.1.211**) using Nuget Manager Window (not command line). 
2. Change the configuration from Any PC to **x86**. 
3. Added the Browser code as seen above. Pressed on Build Project.
4. Got the prompt "Build Failed with one or more errors".

So as an attempt to resolve the issue. Reduced the framework version to 4.5.2 and tried rebuilding it, but got same results.
In order to give it another shot, I downgraded the CefSharp version to 91.1.16 using Nuget manager window and tried rebuilding, but yet again same Strong name issue was prompted.

The project has no Strong Naming turned on. Its a fresh WPF .Net Framework template from Visual Studio.    

Hope this helps :) 


**Update**
Tried creating a new project with 4.5.2 pre selected and it worked. Guess somethings going wrong if you by default create with later versions and then step down.

**Reply by amaitland** [reply]
Sounds like there is a problem with Visual Studio or msbuild. Have you tested on a different machine to see if the problem is specific to your environment? 

I've not seen any other reports of problems.
